### PR TITLE
Учебный проект: начинаем программировать

### PR DESCRIPTION
**Pull request by a201150209**
---
:mortar_board: [Учебный проект: начинаем программировать](https://up.htmlacademy.ru/javascript/17/user/83947/tasks/2)

:boom: https://htmlacademy-javascript.github.io/83947-code-and-magick-17/1/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,187 +1,200 @@
...
+        </div>
+    </template>
+
+    <script>
+        var fireballSize = 22;
+        var getFireballSpeed = function(left) {
+            if (left) {
+                return 5;
+            };
+            return 2;
```
Comment (rkhaslarov) [inline index.html]: please use ternary operator here

File: index.html [hunk 2]
```diff
@@ -1,187 +1,200 @@
...
+        };
+        var wizardSpeed = 3;
+        var wizardWidth = 70;
```
Comment (rkhaslarov) [inline index.html]: vars should on the top of the script.

File: index.html (hunk 1 above)
Comment (a201150209) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (a201150209) [reply index.html]: Done

File: index.html [hunk 3]
```diff
@@ -1,187 +1,197 @@
...
+            return left ? 5 : 2;
+        };
+        var getWizardHeight = function() {
+            return 1.337 * wizardWidth;
+        };
+        var getWizardX = function(width) {
+            return (width - wizardWidth) / 2;
+        };
+        var getWizardY = function(height) {
+            return height / 3 * 2;
```
Comment (rkhaslarov) [inline index.html]: I think, we should just subtract getWizardHeight here, because for the moment your wizard is too close to the earth

Comment (a201150209) [reply index.html]: Done

**Pull request merged**